api/security/api-key-authorize.js

```diff
@@ -6,7 +6,7 @@ const logger = require('../../config/logger')
 const util = require('../util/util')
 
 module.exports = function authorize (request, response, next) {
-  validate(request, function (error, availableScopes) {
+  validate(request, function (error, availableScopes, isSuperAdmin) {
```

Comment:
You should make these changes to the JWT authorizer since that will be the actual use case for super users. Whether or not we leave the api-key able to have a super user as well for easier testing, I will leave up to you. However, once we create the front-end for this, it should be easy to test anyway, without having the api-key table/authorizer changed.